main.py

```diff
@@ -0,0 +1,180 @@
+import decimal
+from decimal import Decimal
+
+# This function adds two numbers
+def add(x, y):
+   return x + y 
+
+# This function subtracts two numbers
+def subtract(x, y):
+    return x - y
+
+# This function multiplies two numbers
+def multiply(x, y):
+    return x * y
+
+# This function divides two numbers
+def divide(x, y):
+    return x / y
+
+# This function modulus two numbers
+def modulus(x, y):
+    return x % y
+
+# This function exponentialize two numbers
+def exponentation(x, y):
+    return(x ** y)
+
+# This function floor divide two numbers
+def floordivide(x, y):
+    return(x // y)
+
+def f():

```

Comment:
```suggestion
def float_calculator():
```
We should use a more descriptive name for this function. What about `float_calculator`?